sympy/printing/tests/test_julia.py

```diff
@@ -212,7 +213,7 @@ def test_containers():
     assert julia_code(Tuple(*[1, 2, 3])) == "(1, 2, 3)"
     assert julia_code((1, x*y, (3, x**2))) == "(1, x.*y, (3, x.^2))"
     # scalar, matrix, empty matrix and empty list
-    assert julia_code((1, eye(3), Matrix(0, 0, []), [])) == "(1, [1 0 0;\n0 1 0;\n0 0 1], zeros(0, 0), Any[])"
```

Comment:
this looks like it introduces backwards incompatibility? If so, I'm afraid it needs a deprecation cycle.